test/CMakeLists.txt

```diff
@@ -247,6 +247,11 @@ if(MIOPEN_TEST_GFX900 OR MIOPEN_TEST_GFX906 OR MIOPEN_TEST_GFX908)
     list(APPEND SKIP_TESTS test_bn_3d_spatial_test)
 endif()
 
+#TODO WORKAROUND_ISSUE_1743: issue with GemmFwdRest
```

Comment:
ref to https://github.com/ROCmSoftwarePlatform/MIOpen/issues/1745
should remove as soon as the CI docker is updated again